go/config/enclave_config.go

```diff
@@ -56,6 +56,8 @@ type EnclaveConfig struct {
 	MessageBusAddress gethcommon.Address
 	// The identity of the sequencer for the network
 	SequencerID gethcommon.Address
+	// Specifies the prefunded addresses at the genesis of the Obscuro network
```

Comment:
Worth mentioning "json string" in this comment imo, so reader understands what to expect.

Reply:
Good point - updated 👍